### PR TITLE
[Feat] - Azure Blob Connector

### PR DESCRIPTION
**Pull request by adriansanchezsequeira-iota**

### CODE REVIEW

File: packages/blob-storage-connector-azure/docs/changelog.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# @twin.org/blob-storage-connector-azure - Changelog
+
+## v0.0.1-next.4
```
Comment (obany) [inline packages/blob-storage-connector-azure/docs/changelog.md:3]: Update to 6

File: packages/blob-storage-connector-azure/src/azureBlobStorageConnector.ts [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+				const chunks: Uint8Array[] = [];
+				for await (const chunk of readableStream) {
+					chunks.push(chunk);
+				}
+
+				const buffer = Buffer.concat(chunks);
+				return new Uint8Array(buffer);
+			}
+			return undefined;
+		} catch {}
```
Comment (obany) [inline packages/blob-storage-connector-azure/src/azureBlobStorageConnector.ts:155]: Probably need to throw an error in here

**Review by obany — CHANGES_REQUESTED** [review]
Couple of changes

File: packages/blob-storage-connector-azure/README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+## Installation
+
+```shell
+npm install @twin.org/blob-storage-connector-azure
+```
+
+## Testing
+
+The tests developed are functional tests and need an instance of Azure up and running. To run Azure locally using local stack:
```
Comment (obany) [inline packages/blob-storage-connector-azure/README.md:13]: this is microsoft container, not local stack

File: packages/blob-storage-connector-azure/README.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+```shell
+docker run -p 10000:10000 -e AZURITE_ACCOUNTS=testAccount:testKey mcr.microsoft.com/azure-storage/azurite azurite-blob --blobHost 0.0.0.0
```
Comment (obany) [inline packages/blob-storage-connector-azure/README.md:16]: Can we add a name parameter to the command to keep it available like with do with the other examples

Comment (adriansanchezsequeira-iota) [reply packages/blob-storage-connector-azure/README.md:16]: I skip that, my bad.

--name azurite is representative enough?

Comment (obany) [reply packages/blob-storage-connector-azure/README.md]: maybe we should come up with a naming scheme for all these images to easily identify them, we can start with this one, something like `twin-blob-azure`

**Review by obany — APPROVED** [review]

**Pull request merged**